MODELGUIDE.md

```diff
@@ -224,7 +224,7 @@ Flow.Ipv4:
       x-field-pattern:
```

Comment:
`random` must be incorporate in the definition of `x-field-pattern` . I am pointing this help section:
```
  features:
    type: string
    enum: [count, auto, metric_group]
    description: >- 
      count:
      Used to specify .....
```
would eb something :
```
  features:
    type: string
    enum: [count, auto, random, metric_group]
    description: >- 
      count:
      Used to specify 
      ....
      random:
      <defined random>
```

Reply:
already mentioned here , will add to openapiart as well
https://github.com/open-traffic-generator/models/pull/380/files#diff-174ecc7450d75d1a117f691cdbe9b658d8843cf43d1177c91548995156a9ab6d

Reply:
ok resolved it. Please note if we extended that document to defined what type of  `properties` we have for `random`. yes, same thing is missing for `increment` and `decrement`.